osbrain/common.py

```diff
@@ -47,6 +50,36 @@ def format_method_exception(error, method, args, kwargs):
     return type(error)(message)
 
 
+def topics_to_bytes(handlers: Dict[Union[bytes, str], Any], uuid: bytes = b''):
+    '''
+    Given some pairs topic/handler, leaves them prepared for making the actual
+    ZeroMQ subscription.
+
+    Parameters
+    ----------
+    handlers
+        Contains pairs "topic - handler".
+    uuid
+        uuid of the SYNC_PUB/SYNC_SUB channel (if applies). For normal
+        PUB/SUB communication, this should be `b''`.
+
+    Returns
+    -------
+    Dict[bytes, Any]
+    '''
+    curated_handlers = {}
+
+    for topic, value in handlers.items():
+        if not isinstance(topic, (str, bytes)):
+            raise TypeError('Incorrect topic type: {}'.format(type(topic)))
```

Comment:
As we are declaring types in the function footprint, I think we could avoid this manual type-check. Specially taking into account that this function is only used internally.

If you'd rather keep that check, just merge it. :wink: